pkg/config/editor_presets.go

```diff
@@ -104,6 +105,15 @@ func standardTerminalEditorPreset(editor string) *editPreset {
 	}
 }
 
+func helixEditorPreset() *editPreset {
+	return &editPreset{
+		editTemplate:              "hx -- {{filename}}",
+		editAtLineTemplate:        "hx -- {{filename}}:{{line}}",
+		editAtLineAndWaitTemplate: "hx -- {{filename}}:{{line}}",
+		editInTerminal:            true,
+	}
+}
+
```

Comment:
All the other presets (except the `standardTerminalEditorPreset` ones) are inlined in the table above, this is the only one that has a separate function. I'd prefer this to be consistent, either inline them all, or make functions for all of them. (I prefer inlining them myself, but not strongly.)

Reply:
Fair points. I've raised a PR for this: https://github.com/jesseduffield/lazygit/pull/2673